client/components/forms/select-opt-groups.jsx

```diff
@@ -1,32 +1,33 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:forms:select-opt-groups' );
+import React, { Component } from 'react';
+import { omit } from 'lodash';
+const debug = require( 'debug' )( 'calypso:forms:select-opt-groups' );
 
-var SelectOptGroups = React.createClass( {
+export default class SelectOptGroups extends Component {
 
-	displayName: 'SelectOptGroups',
-
-	componentWillMount: function() {
+	componentWillMount() {
 		debug( 'Mounting SelectOptGroups React component.' );
```

Comment:
I question the value of these debugging statements. Removing it would also allow us to rewrite this as a stateless function component.